src/library/scala/util/Either.scala

```diff
@@ -457,6 +457,35 @@ object Either {
   def cond[A, B](test: Boolean, right: => B, left: => A): Either[A, B] =
     if (test) Right(right) else Left(left)
 
+  /**
+   * Creates `FromLeftPartiallyApplied[B]` that given an argument of type `A`
+   * produces a `Left` typed as `Either[A,B]`
+   * {{
+   *   val l = Either.left[Int]("boo")
+   *   //l: Either[String, Int] = Left("boo")
+   * }}
+   */
+  def left[B]: FromLeftPartiallyApplied[B] = new FromLeftPartiallyApplied[B]
+
+  private[util] final class FromLeftPartiallyApplied[B](private val dummy: Boolean = true) extends AnyVal {
```

Comment:
I think these `dummy`s can now be `Unit` since 2.12.3 :)

Reply:
https://github.com/scala/scala/commit/34da49bc31b4ba15d385d3c6112808255d2de2ef